rs/ethereum/ledger-suite-orchestrator/tests/tests.rs

```diff
@@ -7,49 +7,21 @@ use ic_ledger_suite_orchestrator::candid::{
     ManagedCanisters, ManagedLedgerSuite, OrchestratorArg, OrchestratorInfo,
     UpdateCyclesManagement, UpgradeArg,
 };
-use ic_ledger_suite_orchestrator_test_utils::arbitrary::{arb_init_arg, arb_principal};
 use ic_ledger_suite_orchestrator_test_utils::{
     assert_reply, cketh_installed_canisters, default_init_arg, ledger_suite_orchestrator_wasm,
-    new_state_machine, supported_erc20_tokens, usdc, usdc_erc20_contract, usdt,
-    LedgerSuiteOrchestrator, GIT_COMMIT_HASH_UPGRADE, MINTER_PRINCIPAL, NNS_ROOT_PRINCIPAL,
+    new_state_machine, usdc, usdc_erc20_contract, usdt, LedgerSuiteOrchestrator,
+    GIT_COMMIT_HASH_UPGRADE, MINTER_PRINCIPAL, NNS_ROOT_PRINCIPAL,
 };
 use ic_state_machine_tests::ErrorCode;
 use icrc_ledger_types::icrc::generic_metadata_value::MetadataValue as LedgerMetadataValue;
 use icrc_ledger_types::icrc1::account::Account as LedgerAccount;
-use proptest::prelude::ProptestConfig;
-use proptest::proptest;
 use std::sync::Arc;
 
 const MAX_TICKS: usize = 10;
 const GIT_COMMIT_HASH: &str = "6a8e5fca2c6b4e12966638c444e994e204b42989";
 
 pub const TEN_TRILLIONS: u64 = 10_000_000_000_000; // 10 TC
 
-proptest! {
-    #![proptest_config(ProptestConfig {
-            cases: 10,
-            .. ProptestConfig::default()
-        })]
-    #[test]
-    fn should_install_orchestrator_and_add_supported_erc20_tokens(mut init_arg in arb_init_arg(), minter_id in arb_principal()) {
```

Comment:
@THLO @ninegua : I have the impression this test is overkill, it takes 25s and tries to test arbitrary well-formed inputs, but the reality is that we are the one doing the proposals to add a new token or at least reviewing community proposals to add such tokens so testing against abitrary inputs seems hardly worth it. The test below `should_spawn_ledger_with_correct_init_args` is realistic enough so that I don't feel the need for additional testing on that matter. WDYT?

Reply:
I agree.

Reply:
Sounds good!